Gruntfile.coffee

```diff
@@ -1169,6 +1198,20 @@ module.exports = (grunt) ->
     'exec:androidReplaceXwalkDist'
   ]
 
+  # Identical to the release build but with monkey-proofing, for use by
+  # automated test systems without contaminating usage metrics.
+  grunt.registerTask 'monkey_proof_android', [
```

Comment:
This will be *three* Android builds. How about making the **dev** build metrics-less rather than introducing another build? Seems like we don't want our own testing to submit metrics, either.